### ISSUE TITLE
Latest Chrome extension does not work

### ISSUE THREAD

**Issue by smaragdus** [erickutcher/httpdownloader#122]
Chrome extension version [1.0.1.5](https://github.com/erickutcher/httpdownloader/releases/tag/CE_v1.0.1.5) and version [1.0.1.6](https://github.com/erickutcher/httpdownloader/releases/tag/CE_v1.0.1.6) do not work for me- the downloads are not passed to [HTTP Downloader](https://erickutcher.github.io/#HTTP_Downloader), they appear in browser download manager as  `Canceled`, screen:

![HTTP_Downloader_Chrome_Extension 1 0 1 6 - 2020-02-20 - 002](https://user-images.githubusercontent.com/9025757/74888467-ad832e80-5386-11ea-8600-03b97b8fc2ac.png)

I had to revert to version [1.0.1.4](https://github.com/erickutcher/httpdownloader/releases/tag/CE_v1.0.1.4) to be able to use browser integration.

**Comment by RatError** [comment]
I confirm that versions 1.0.1.5 and 1.0.1.6 do not work.

**Comment by erickutcher** [comment]
This is when it's set to override the browser's download manager?

**Comment by smaragdus** [comment]
@erickutcher 

> This is when it's set to override the browser's download manager?

Yes, 'Override browser's download manager' is almost always on, I disable it on very rare occasions when for some reason HTTP Downloader cannot get a file.

> Edit: I figured it out. I'll have a new version up this weekend.

Thanks. Didn't you notice this problem or it does happen only under special circumstances?

**Comment by erickutcher** [comment]
I don't override the browser's download manager and normally just right click links or grab it from the browser's network inspector.

The problem is how these browsers give you cookies. You can request them using a full URL or a domain, but depending on which one you use, you may not get everything. So you end up having to do these goofy workarounds just to get what you need. In this case I wasn't checking to make sure that the returned value was actually an array. It's an easy fix, but I want to look into it a little more.

**Comment by erickutcher** [comment]
A new version is up. Should be fixed now.

**Comment by smaragdus** [comment]
@erickutcher 

I tested both [Chrome extension version 1.0.1.7](https://github.com/erickutcher/httpdownloader/releases/tag/CE_v1.0.1.7) and [Firefox extension version 1.0.1.7](https://github.com/erickutcher/httpdownloader/releases/tag/FWE_v1.0.1.7) and I confirm that both work fine. When I reported this issue I had tested only the Chrome extension so I did not know whether the Firefox extension was also affected. Thank you for the new release (1.0.1.7) of the extensions, I am closing this issue as the problem seems to be fixed.
